docs/contents/community/faq.md

```diff
@@ -54,4 +54,4 @@
 
 !!! quote "Answer"
     Excellent question and we're super excited that you're interested in ACK.
-    For now, if you're a developer, you can check out the [contributor docs](../../dev-docs/overview/).
\ No newline at end of file
+    For now, if you're a developer, you can check out the [contributor docs](../contents/dev-docs/overview/).
```

Comment:
Nope, the previous link (`../../dev-docs/overview/`) is the correct one. Please always check with `mkdocs serve` locally. Side note to myself: we really should make link checker part of the CI.

Reply:
I was previewing it locally which is why I was surprised the 2nd comment said the link was broken. I realized I fix that but forgot to save/commit/push so when I saw the comment I unfixed it and push it broken.

looks like with `strict: true` in the mkdocks.yaml we could error if there were warnings for things like broken links but that may cause other friction for small warnings.

Reply:
Gotcha, thanks for the clarification and that's a great suggestion. Will play around with it and see if we can use it.